### PR TITLE
[Improve][E2E] Improve testcase output log style

### PR DESCRIPTION
**Pull request by hailin0**
## Purpose of this pull request

Improve testcase output log style

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: seatunnel-e2e/seatunnel-flink-connector-v2-e2e/connector-mongodb-flink-e2e/src/test/java/org/apache/seatunnel/e2e/flink/v2/mongodb/MongodbIT.java [hunk 1]
```diff
@@ -90,7 +91,7 @@ public void startMongoContainer() {
                 .forPort(MONGODB_PORT)
                 .forStatusCodeMatching(response -> response == HTTP_OK || response == HTTP_UNAUTHORIZED)
                 .withStartupTimeout(Duration.ofMinutes(2)))
-            .withLogConsumer(new Slf4jLogConsumer(log));
+            .withLogConsumer(new Slf4jLogConsumer(DockerLoggerFactory.getLogger(MONGODB_IMAGE)));
```
Comment (hailin0) [inline seatunnel-e2e/seatunnel-flink-connector-v2-e2e/connector-mongodb-flink-e2e/src/test/java/org/apache/seatunnel/e2e/flink/v2/mongodb/MongodbIT.java:94]: log example:

image

File: seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/AbstractTestContainer.java [hunk 2]
```diff
@@ -102,8 +102,18 @@ protected Container.ExecResult executeCommand(GenericContainer<?> container, Str
...
+                    + "{}"
+                    + "\n==================== ExecuteConfigFile: {} STDOUT end   ====================",
+                configPath, execResult.getStdout(), configPath);
+        }
+        if (execResult.getStderr() != null && execResult.getStderr().length() > 0) {
+            LOG.error("\n==================== ExecuteConfigFile: {} STDERR start ====================\n"
+                    + "{}"
+                    + "\n==================== ExecuteConfigFile: {} STDERR end   ====================",
+                configPath, execResult.getStderr(), configPath);
+        }
```
Comment (hailin0) [inline seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/AbstractTestContainer.java:116]: log example

image

**Comment by hailin0** [comment]
network timeout

image

**Comment by TyrantLucifer** [comment]
> network timeout
> 
> image

Restarted yet.

**Comment by hailin0** [comment]
waiting for #3031  fixd
image

**Review by Hisoka-X — APPROVED** [review]
LGTM

**Review by EricJoy2048 — APPROVED** [review]
LGTM

**Pull request merged**